models/shipment.js

```diff
@@ -11,6 +11,19 @@ module.exports = (sequelize, DataTypes) => {
             group: {
                 type: DataTypes.STRING,
                 allowNull: false
+            },
+            contact_email: {
+                type: DataTypes.STRING,
+                allowNull: false,
+                validate: {
+                    isValidEmail: val => {
+                        let regexp = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,5})+$/
```

Comment:
do we want to make sure it's a @turner.com?

Reply:
Is that a requirement?  That's not specified on official policy page.